pandas/core/generic.py

```diff
@@ -4287,6 +4287,38 @@ def _clip_with_scalar(self, lower, upper, inplace=False):
         else:
             return result
 
+    def _clip_with_one_bound(self, threshold, method, axis, inplace):
+
+        if np.any(isnull(threshold)):
+            raise ValueError("Cannot use an NA value as a clip threshold")
+
+        # method is self.le for upper bound and self.ge for lower bound
+        if is_scalar(threshold) and is_number(threshold):
+            if method.__name__ == 'le':
+                return self._clip_with_scalar(None, threshold, inplace=inplace)
+            else:
+                return self._clip_with_scalar(threshold, None, inplace=inplace)
+
+        inplace = validate_bool_kwarg(inplace, 'inplace')
```

Comment:
Thanks noticing this. I fixed it.

I actually realized this as well. This inplace is kind of validated twice not not validated, if the call is form `clip_upper` with scalar and number threshold. However, I don't know if I should move the check form `clip` to `_clip_with_scalar` or the other way around. I just left it as is.